src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
+     */
+    @Setter
+    @Getter
+    @ToString
+    @Data
```

Comment:
Using Lombok's @Data annotation can lead to violation of the Single Responsibility Principle (SRP) as it generates getters, setters, equals, hashCode, and toString methods, which might not be necessary for all attributes. (Severity: medium)